src/opentimelineview/clip_inspector_widget.py

```diff
@@ -0,0 +1,208 @@
+            if self.range_type == 1:
+                self.rangeComboBox.setCurrentIndex(0)
+        else:
+            self.rangeComboBox.model().item(1).setEnabled(True)
+        if clip.available_range() is None:
+            self.rangeComboBox.model().item(2).setEnabled(False)
+            if self.range_type == 2:
+                self.rangeComboBox.setCurrentIndex(0)
+        else:
+            self.rangeComboBox.model().item(2).setEnabled(True)
+        self.videoLayout.setCurrentIndex(0)  # show video player widget
+        self.player.stop()  # stop player before changing media state
+        if isinstance(clip, otio.schema.Clip):
+            self.clip = clip
+            if len(clip.effects) != 0:  # show effects banner if effects present
+                effectsString = " | "
+                effectsString = effectsString.join(
+                    effect.effect_name for effect in clip.effects)
+                self.effectsLabel.setText("Effects: " + effectsString)
+            else:
+                self.effectsLabel.setText("Effects: ")
+            path = clip.media_reference.target_url
+            if path.startswith('file://'):
+                path = path[7:]
+                self.player.setMedia(QUrl.fromLocalFile(path))
+            elif path.startswith('http'):
+                self.player.setMedia(QUrl(path))
+
+    def play_clip(self):
+        self.player.play()
+
+    def pause_clip(self):
+        self.player.pause()
+
+    def stop_clip(self):
+        self.player.stop()
+
+    def set_clip_position(self, position):
+        self.player.setPosition(position)
+
+    def create_error_images(self):
+        # painter for unresolved media error image
+        painter = QPainter(self.unresolvedMediaErrorImage)
+        painter.setBrush(QBrush(Qt.green))
+        painter.fillRect(QRectF(0, 0, self.clipWidth, self.clipHeight), Qt.green)
+        painter.setPen(QPen(Qt.black))
+        # use larger font size
+        font = painter.font()
+        font.setPointSize(font.pointSize() * 3)
+        painter.setFont(font)
+        painter.drawText(QRectF(self.clipWidth * 0.25, self.clipHeight * 0.25,
+                                self.clipWidth * 0.5, self.clipHeight * 0.6),
+                         "Media Reference can't be resolved.")
+        # painter for unsupported media error image
+        painter = QPainter(self.unsupportedMediaErrorImage)
+        painter.setBrush(QBrush(Qt.green))
+        painter.fillRect(QRectF(0, 0, self.clipWidth, self.clipHeight), Qt.green)
+        painter.setPen(QPen(Qt.black))
+        painter.setFont(font)
+        painter.drawText(QRectF(self.clipWidth * 0.25, self.clipHeight * 0.25,
+                                self.clipWidth * 0.5, self.clipHeight * 0.6),
+                         "Unsupported Media format.")
+
+    def handle_error(self):
+        if self.player.errorString() == 'Resource not found.' or \
+                self.player.errorString() == 'Not Found':
```

Comment:
Its usually more readable to use `()` to do line continuation:
```suggestion
        if (
                self.player.errorString() == 'Resource not found.' 
                or self.player.errorString() == 'Not Found'
        ):
```

That way the or also comes first, showing the logical relationship of the two conditions.